Azure-Samples/enterprise-data-sharing
# fix: add sp creation to prerequirements

## Purpose
Adding graph_sp to prerequirements
Adding changes to ReadMe

## Does this introduce a breaking change?
[ ] Yes
[X] No
```

## Pull Request Type
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```
## How to Test
*  Run prerequirments.sh
* Check the graph_sp creation in Entra ID
